luigi_pipeline/seqr_loading_optimized.py

```diff
@@ -1,36 +1,31 @@
 import logging
-import os
 import sys
-import re
 
 import luigi
 import hail as hl
 
-from lib.hail_tasks import HailMatrixTableTask, HailElasticSearchTask, GCSorLocalTarget, MatrixTableSampleSetError
-from lib.model.seqr_mt_schema import SeqrSchema, SeqrVariantSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
-import seqr_loading
+from lib.hail_tasks import HailMatrixTableTask, HailElasticSearchTask
+from lib.model.seqr_mt_schema import SeqrVariantSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+from seqr_loading import SeqrVCFToMTTask
 
 logger = logging.getLogger(__name__)
 
 
-class SeqrVCFToVariantMTTask(seqr_loading.SeqrVCFToMTTask):
+class SeqrVCFToVariantMTTask(SeqrVCFToMTTask):
     """
     Loads all annotations for the variants of a VCF into a Hail Table (parent class of MT is a misnomer).
     """
-
-    def run(self):
```

Comment:
this is still needed, the optimized class absolutely needs a different `run`, it should just now be 
```
def run(self):
    # We only want to use the Variant Schema.
    self.read_input_write_mt()
```

Reply:
The difference compared with the inherited `run()` is that it doesn't check the input files exist. Why don't we need those checks?

Reply:
please do not change the behavior of the optimized task we use for all or data loading, we made it different on purpose

Reply:
maintaining this functionality blocks approval